DailyReportWeb/Helpers/PolicyService.cs

```diff
@@ -0,0 +1,40 @@
+﻿using SourceControlLogReporter.Model;
+using System;
+using System.Collections.Generic;
+using System.Configuration;
+using System.Linq;
+using System.Web;
+
+namespace DailyReportWeb.Helpers
+{
+    public class PolicyService
+    {
+        public static void SetPolicyFullDraft(Policy policy, string policyPath)
+        {
+            policy.AdvancedOptions.NoIndividualDraft = true;
+            policy.AdvancedOptions.NoDraft = false;
+            policy.GeneratedProperties.IndividualDrafts = null;
+            policy.SaveToFile(policyPath);
+        }
+
+        public static void SetPolicyFinalReport(Policy policy, string policyPath)
+        {
+            policy.AdvancedOptions.NoDraft = true;
```

Comment:
@Sebyd 
Nu e ok sa lucrezi cu proprietatile astea din AdvancedOptions. Astea sunt puse in policy si reprezinta setarile pentru acel proiect, daca vrei sau nu draft si daca vrei sau nu draft individual.

Pentru ca sa iti marchezi ca s-a trimis deja draftul repectiv ca s-au trimis deja drafturile individual trebuie sa iti pui niste variabile noi in GenertatedProperties. 

Altfel nu imi dau seama cum functioneaza proprietatile astea daca vrei sa modifici policy-ul cand tu le folosesti pentru a marca diferite chestii.